### PR TITLE
fix(fx-2517): set default sort to "gallery curated"

### PR DESCRIPTION
**Pull request by dzucconi**
Will open this in draft. If you think this the right approach to take, I'll add some specs here.

I kind of stumbled through this — there may be (I hope) an easier/more straightforward way to do this.

--------

The type of this PR is: **Bugfix**

This PR resolves [FX-2517]

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[FX-2517]: https://artsyproduct.atlassian.net/browse/FX-2517

### CODE REVIEW

File: src/lib/utils/ArtworkFilter/ArtworkFiltersStore.tsx [hunk 1]
```diff
@@ -303,18 +301,24 @@ export const selectedOptionsUnion = ({
...
+      paramName: FilterParamName.sort,
+      paramValue: "position",
+      displayText: "Lot number ascending",
+    },
+    showArtwork: {
+      paramName: FilterParamName.sort,
+      paramValue: "partner_show_position",
+      displayText: "Gallery Curated",
+    },
+  }[filterType]
```
Comment (dzucconi) [inline src/lib/utils/ArtworkFilter/ArtworkFiltersStore.tsx:320]: This way of assignment is much easier to deal with in Typescript. Even if there are only two options, adding additional filterTypes would give you type errors everywhere you need to update.

**Review by anandaroop — APPROVED** [review]
LGTM

File: src/lib/utils/ArtworkFilter/ArtworkFiltersStore.tsx [hunk 2]
```diff
@@ -251,10 +252,7 @@ export const reducer = (
 }
 
 const getSortDefaultValueByFilterType = (filterType: FilterType) => {
-  if (filterType === "saleArtwork") {
-    return "position"
-  }
-  return "-decayed_merch"
+  return { artwork: "-decayed_merch", saleArtwork: "position", showArtwork: "partner_show_position" }[filterType]
```
Comment (MounirDhahri) [inline src/lib/utils/ArtworkFilter/ArtworkFiltersStore.tsx:255]: 👏

File: src/lib/utils/ArtworkFilter/ArtworkFiltersStore.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/lib/utils/ArtworkFilter/ArtworkFiltersStore.tsx:320]: I agree 👏

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
👏👏👏 LGTM, I will assign for the PR and feel free to merge once it's ready for review

**Pull request merged**